docs/network/migration-guide.md

```diff
@@ -174,7 +174,7 @@ Install the `@lit-protocol/lit-node-client` package, which can be used in both b
 yarn add @lit-protocol/lit-node-client@beta
```

Comment:
i think we can remove the beta tag now right? It is no longer included in the 'installation' section